themes/upbound/assets/scss/_hints.scss

```diff
@@ -68,7 +69,10 @@ $hint-note-background:          rgba(0,176,255,.1);
 //
 
 .admonition {
-    border-left: .3rem solid;
+    border-radius: 0.375rem;
+    border: 1px solid rgba(205,205,211, 1);
+    opacity: 1;
+    background-color: rgba(248,249,249, 1);
 
     &.warning {
         border-left-color: #{$hint-warning};
```

Comment:
Warnings are styled differently now than the other boxes by maintaining their title background bar, not sure if that was on purpose.